content/docs/data/Zapier.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Zapier'
+meta_title: 'Appbase - Zapier'
+meta_description: 'Automate your appbase.io database operations with Zapier.'
+keywords:
+    - dataschema
+    - appbase
```

Comment:
@ShahAnuj2610 Replace all usage of `appbase` and `Appbase` with `appbase.io` instead.